packages/js/client-config/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Let's add a some description here

Reply:
Added a slightly modified version of the description found in this hackmd I'm setting up:
https://hackmd.io/0rDyONfoT4mw2HxTYtjAxA

Hope it looks good!